chart/templates/create-user-job.yaml

```diff
@@ -57,14 +57,6 @@ spec:
         - name: {{ template "registry_secret" . }}
       {{- end }}
       containers:
-        - name: run-airflow-migrations
```

Comment:
Should we add wait-for-airflow-migrations here? 
https://github.com/apache/airflow/blob/7b7cc3c391d756e0ef55df8f0c674da4de3fb4f0/chart/templates/webserver/webserver-deployment.yaml#L78

Reply:
@mik-laj I don't think it matters as this is a job, so it'll keep failing until it passes.

Reply:
I am still thinking about the tests. Should we add any tests? My heart hurts when it adds code that does not have automatic or semi-automatic tests. Is it possible to add some automated tests here so that future contributors can verify our assumptions?

Reply:
@mik-laj this code is already tested. If the migrations don't happen, the k8s tests never launch.

Reply:
The code is under high-level testing, but that doesn't mean we shouldn't think about lower-level testing. I don't have much experience testing Helm Chart yet, but if you change the behavior of the code, the tests should also be updated. 

I am able to imagine that our test could consist in generating the YAML code and checking only the type and name of the resource. That would be enough because we would know that these resources are really being created.